gtfs-realtime/spec/en/reference.md

```diff
@@ -439,14 +439,14 @@ TripDescriptor.route_id cannot be used within an Alert EntitySelector to specify
-| **schedule_relationship** | [ScheduleRelationship](#enum-schedulerelationship-1) | Optional | One | The relation between this trip and the static schedule. If TripDescriptor is provided in an Alert `EntitySelector`, the `schedule_relationship` field is ignored by consumers when identifying the matching trip instance.
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|------------------|------------|----------------|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
These changes are unrelated to the clarification. Can you prevent them in the future?

Reply:
There are some changes here in the text related to the clarification. GitHub doesn't always do great rendering diffs of tables. Try clicking on the "Display the rich diff" button on the referenced.md file and you should see an easier to read version (see red circle in image for button, and strikethrough change starting at the bottom):

![image](https://user-images.githubusercontent.com/928045/156855390-13552999-405c-4cb8-bbeb-08804e4158f5.png)

Reply:
Oh, maybe you're referring to the expanded table header? If so, yes, not sure why that happened. Maybe artifact of the editor I was using for markdown. Will try to avoid next time.

Reply:
I reverted the table header but I'm fighting IntelliJ to fix the other line endings. I'll work on reverting the changes to the pipe characters on the remaining lines early next week.

Reply:
Ok, I believe I've removed the unrelated formatting changes.